PSPDFKitPlugin/PSPDFKitPlugin.m

```diff
@@ -1562,6 +1561,14 @@ - (void)applyInstantJSON:(CDVInvokedUrlCommand *)command {
     return [annotationsJSON copy];
 }
 
+- (PSPDFAnnotationType)annotationTypeFromInstantJSON:(NSString *)typeString {
```

Comment:
Since this method not only converts Instant JSON annotation types, but also strings to annotation types, I would use a more general name.

```suggestion
- (PSPDFAnnotationType)annotationTypeFromString:(NSString *)typeString {
```